### PR TITLE
Rename job history and convert the button to a link to a new tab

### PR DESCRIPTION
**Pull request by superstar54**
Fix https://github.com/aiidalab/aiidalab-qe/issues/996 and https://github.com/aiidalab/aiidalab-qe/issues/997

- Rename `Job history` to `Calculation history`
- Convert the button to a link to a new tab

### CODE REVIEW

File: src/aiidalab_qe/app/wrapper.py [hunk 1]
```diff
@@ -275,3 +249,9 @@ def __init__(self) -> None:
                 footer,
             ],
         )
+
+    def _open_calculation_history(self, _):
+        from IPython.display import Javascript
+
+        url = "./calculation_history.ipynb"
+        display(Javascript(f"window.open('{url}', '_blank')"))
```
Comment (edan-bainglass) [inline src/aiidalab_qe/app/wrapper.py:257]: See #988. I think this will run into a similar issue.

**Review by edan-bainglass — CHANGES_REQUESTED** [review]
Thanks @superstar54. Looks good and works on my end. A few comments:

- We'll have to figure out another approach for linking to another notebook to avoid browser pop-up blocker (#988)

- Need a space between Quantum and ESPRESSO in the notebook title
- Could we make the table expand to full page width?

image

**Comment by superstar54** [comment]
Hi @edan-bainglass , thanks for the review.

> Could we make the table expand to full page width?

Do you know how to do this? I tried `ipw.Layout(width="100%")`, but it does not work.

**Review by edan-bainglass — APPROVED** [review]
Excellent. LGTM! Thanks @superstar54. I'll look into the pop-up blocker issue today. Once I have a fix, I can also apply it to your button.

**Pull request merged**